R-package/src/lightgbm_R.h

```diff
@@ -564,34 +557,28 @@ LIGHTGBM_C_EXPORT SEXP LGBM_BoosterSaveModel_R(
 
 /*!
 * \brief create string containing model
-* \param handle handle
+* \param handle Booster handle
 * \param num_iteration, <= 0 means save all
-* \param out_str string of model
-* \return 0 when succeed, -1 when failure happens
+* \param feature_importance_type type of feature importance, 0: split, 1: gain
+* \return length-1 R character vector with model string
```

Comment:
What does mean `length-1`?..

Reply:
```suggestion
* \return R character vector (length=1) with model string
```

Is this re-write clearer? It's common when talking about R to say `length-n vector` to mean "A vector with `n` elements", but if that's not clear to you it probably will be unclear to others reading this code.

This is one of those places where the fact that R doesn't have scalar types makes the language a little awkward. In most other languages we'd just say this is a string.

Reply:
I updated to this, but will leave the thread unresolved in case you have a different suggestion

Reply:
I tried to find a variable named `length` described somewhere earlier in the description and figure out why this vector contains `length-1` ("`length minus 1`") elements. 😄